apiserver/HACluster.md

```diff
@@ -39,8 +39,8 @@ kubectl apply -f <your location>/kuberay/apiserver/test/cluster/redis/redis_pass
 
 ## Ray Code for testing
 
-For both Ray jobs and Ray Serve we recommend packaging user code in the image. For a simple testing here
-we will create a [config map](test/cluster/code_configmap.yaml), containg simple code, that we will use for
+For both Ray Jobs and Ray Serve we recommend packaging user code in the image. For a simple testing here
+we will create a [config map](test/cluster/code_configmap.yaml), containging simple code, that we will use for
```

Comment:
"containging" should be probably _containing_

Reply:
fixed